AmplifyPlugins/Storage/AWSS3StoragePluginFunctionalTests/AWSS3StoragePluginTestBase.swift

```diff
@@ -11,143 +11,33 @@ import AWSMobileClient
+@testable import AmplifyTestCommon
 
 class AWSS3StoragePluginTestBase: XCTestCase {
 
-    /*
-     Set up
-     `amplify init`
-     `amplify add storage`
-     ? Please select from one of the below mentioned services: `Content (Images, audio, video, etc.)`
-     ? You need to add auth (Amazon Cognito) to your project in order to add storage for user files. Do you want to add auth now? `Yes`
-      Do you want to use the default authentication and security configuration? `Default configuration`
-      How do you want users to be able to sign in? `Username`
-      Do you want to configure advanced settings? `No, I am done.`
-     Successfully added auth resource
-     ? Please provide a friendly name for your resource that will be used to label this category in the project: `s3f34a5918`
-     ? Please provide bucket name: `storage6e54d3cf8ff042e18d59639bdb4f1664`
-     ? Who should have access: `Auth and guest users`
-     ? What kind of access do you want for Authenticated users? `create/update, read, delete`
-     ? What kind of access do you want for Guest users? `create/update, read, delete`
-     ? Do you want to add a Lambda Trigger for your S3 Bucket? `No`
-     `amplify push`
+    static let amplifyConfiguration = "AWSS3StoragePluginTests-amplifyconfiguration"
+    static let awsconfiguration = "AWSS3StoragePluginTests-awsconfiguration"
 
-     awsconfiguration.json
-     {
-         "UserAgent": "aws-amplify/cli",
-         "Version": "0.1.0",
-         "IdentityManager": {
-             "Default": {}
-         },
-         "CredentialsProvider": {
-             "CognitoIdentity": {
-                 "Default": {
-                     "PoolId": "us-west-2:xxx",
-                     "Region": "us-west-2"
-                 }
-             }
-         },
-         "CognitoUserPool": {
-             "Default": {
-                 "PoolId": "us-west-2_xx",
-                 "AppClientId": "xxx",
-                 "AppClientSecret": "xxx",
-                 "Region": "us-west-2"
-             }
-         },
-         "S3TransferUtility": {
-             "Default": {
-                 "Bucket": "xxx-devo",
-                 "Region": "us-west-2"
-             }
-         }
-     }
-
-     amplifyconfiguration.json
-     {
-         "UserAgent": "aws-amplify-cli/2.0",
-         "Version": "1.0",
-         "storage": {
-             "plugins": {
-                 "awsS3StoragePlugin": {
-                     "bucket": "xx-devo",
-                     "region": "us-west-2",
-                     "defaultAccessLevel": "guest"
-                 }
-             }
-         }
-     }
-     */
-    let bucket: JSONValue = "xxxxxxx-devo"
-    let region: JSONValue = "us-east-1"
-    let networkTimeout = TimeInterval(180) // 180 seconds to wait before network timeouts
-    static let largeDataObject = Data(repeating: 0xff, count: 1_024 * 1_024 * 6) // 6MB
+    static let largeDataObject = Data(repeating: 0xff, count: 1_024 * 1_024 * 6) // 6MBb
```

Comment:
What's an "MBb"? Did you mean to specify Mebibyte "MiB"?

Reply:
haha that was `6MB`, and when i press CMD+B but my CMD button failed me, reverting change